epictrack-web/src/components/indigenousNation/IndigenousNationForm.tsx

```diff
@@ -154,29 +164,37 @@ export default function IndigenousNationForm({ ...props }) {
           </Grid>
           <Grid item xs={6}>
             <ETFormLabel>PIP Organization Type</ETFormLabel>
-            <ControlledSelectV2
-              placeholder="Select an Organization Type"
-              defaultValue={(ctx.item as FirstNation)?.pip_org_type_id || ""}
-              getOptionLabel={(o: PIPOrgType) => (o ? o.name : "")}
-              getOptionValue={(o: PIPOrgType) => (o ? o.id.toString() : "")}
-              options={pipOrgTypes || []}
-              {...register("pip_org_type_id")}
-            ></ControlledSelectV2>
+            <div data-cy="indigenous-form-pip-organization-type">
```

Comment:
Instead of having a separate div for the data-cy, can you pass it to the ControlledSelectV2 component and set it in the underlying dom element

Reply:
just tried doing this but the inner dom element controlled and track select still dont recognize the data-cy so i wrapped the inner element with a div instead let me know if thats good